client/src/Home.js

```diff
@@ -1,9 +1,15 @@
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
 import { Link } from 'react-router-dom';
 import './Home.scss';
+import Api from './Api';
 
 function Home() {
   const [active, setActive] = useState(false);
```

Comment:
@JesusEArambula I think I mentioned this on the first PR... but this `active` state variable you copied from Header.js is not doing anything in this file (it was used to control whether the drop-down menu was open or not in the Header).

So, you can remove this line, and remove all the `onClick` props in the Link tags below....

Reply:
I removed Api, but now the cards aren't showing up, and VS Code gives me an error for removing the onClick prop from the Link tag.
I thought we needed to import Api for the map() function on line 25.

Reply:
You DO need to import Api. 

My comment was anchored on line 7 and was talking about the `active`/`setActive` state variable and function created by useState. This line can be removed because the `active` state variable is not being used for anything in this file- you copied from Header.js where it was being used to open/close the drop-down menu.

What is the error you're getting from removing the onClick props on the LInk tags?

Reply:
Got it.
Also, the onClick error is gone.